htdocs/reports/reports_testhistory.php

```diff
@@ -960,8 +985,8 @@ function change_orientation() {
 
 <br/><br/>
 <?php $align=$report_config->alignment_header;?>
-<h3 align="<?php echo $align; ?>"><?php echo $report_config->headerText; ?><?php #echo LangUtil::$pageTerms['MENU_PHISTORY']; ?></h3>
-<h4 align="<?php echo $align; ?>"><?php echo $report_config->titleText; ?></h4>
+<h3 id="lab_header" align="<?php echo $align; ?>"><?php echo $report_config->headerText; ?><?php #echo LangUtil::$pageTerms['MENU_PHISTORY']; ?></h3>

```

Comment:
```suggestion
<h3 id="lab_header" align="<?php echo $align; ?>"><?php echo $report_config->headerText; ?></h3>

```

Reply:
Good catch, this must have been leftover from before. I didn't even realize it was commented out.